tracee-rules/signatures/rego/anti_debugging_ptraceme.rego

```diff
@@ -1,6 +1,7 @@
-package main
+package anti_debugging_ptraceme
```

Comment:
Note: I have called it `anti_debugging_ptraceme` rather than `tracee.TRC-2` this is because dashes don't seem to be allowed characters in rego package names https://www.openpolicyagent.org/docs/latest/policy-language/#packages

I thought it's just easier to rely on the filename here since filenames will always have to be unique.

Reply:
I _could_ also call it `tracee.TRC2`, fine either way I believe.

Reply:
thanks for spotting this. Not that I'm suggesting this, just an allegory: If you want to use filename here, why not use filename for ID as well?
I tend to think we should stick with TRC2/TRC_2, and for consistency, rename the files as well. wdyt?

Reply:
updated